doc/examples/plot_circular_elliptical_hough_transform.py

```diff
@@ -128,10 +128,10 @@
 
 # Estimated parameters for the ellipse
 best = result[-1]
```

Comment:
Changing this line to `result[-1].round()` would be a more minimal way to accomplish this. Actually, this example would probably be best if we went further:

`best = result[-1].round().astype(int)`

and removed the repeated `int()` calls.